app/src/main/AndroidManifest.xml

```diff
@@ -512,6 +512,10 @@
             </intent-filter>
         </receiver>
 
+        <receiver
+            android:name=".remotemessage.SharePromoLinkRMFBroadCastReceiver"
+            android:exported="false" />
+
```

Comment:
Does this need to be inside the :app? seems to belong to the RMF library as it's an action to an RMF (?)

Reply:
I think so, the receiver is called from the :app module. The examples I've seen where we implement the BroadcastReceiver within the module are when it's called within the :impl feature module.